apps/admin/src/app/page.tsx

```
@@ -0,0 +1,64 @@
+"use client";
+
+import { useQuery } from "convex/react";
+import { api } from "@buzztrip/backend/api";
+import { StatCard } from "@/components/dashboard/stat-card";
+import { MapsChart } from "@/components/charts/maps-chart";
+import { MarkersChart } from "@/components/charts/markers-chart";
+import { Users, Map, MapPin, MapPinned } from "lucide-react";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)